scanner/src/main/kotlin/scanners/fossid/FossId.kt

```diff
@@ -170,6 +170,7 @@ class FossId(
         user = fossIdScannerOptions["user"]
             ?: throw IllegalArgumentException("No FossId User configuration found.")
         packageNamespaceFilter = fossIdScannerOptions["packageNamespaceFilter"].orEmpty()
+        packageAuthorsFilter = fossIdScannerOptions["packageAuthorsFilter"].orEmpty()
```

Comment:
Is this just a plain string referencing a single author? Then this would be rather limited. Should it be at least a comma-separated list? (Although this would introduce the problem if an author name can contain a comma.)

Reply:
I wanted to keep things simple.
The standard use case if you have a project with it's dependencies, and you want to only scan the project, i.e. there is only one author.
We used this option for several projects and we never faced this as a limitation.

Reply:
Ok, fine with me. As long as the feature is not needed, we do not have to complicate things.